City-of-Helsinki/berth-reservations-admin
# VEN-651 | Show harbor card on offer page

## Description :sparkles:

* Show harbor card on offer page
* Turn HarborProperties into common HarborCard component, along with Property
  * Add tests and stories for HarborCard
  * Update HarborCard and PierProperties to use terms from `common.terminology`
* Move HarborViewPage to harborView for consistency between features
  * Move HarborViewPage tests to `__tests__`, use TS

## Issues :bug:

### Closes :no_good_woman:

* [VEN-651](https://helsinkisolutionoffice.atlassian.net/browse/VEN-651): FE - Offer page: Harbor card

### Related :handshake:

## Testing :alembic:

### Automated tests :gear:️

* Added tests for HarborCard
* OfferPage has no tests... 🌧️

### Manual testing :construction_worker_man:

* Open offer page and check that the harbor card is visible and contains correct data

## Screenshots :camera_flash:

## Additional notes :spiral_notepad:
